### ISSUE TITLE
Add alt text to lightbox images

### ISSUE THREAD

**Issue by erinchambers** [CDRH/bandhistory_rails#47]
Splitting this issue out of #44 - alt text is missing for gallery/lightbox images. These are overridden from datura and orchid so changes will need to be made here and in the data repo.

**Comment by techgique** [comment]
I've implemented these changes on a `fancybox` branch and checked it out on cdrhrailsdev.

But it changes the UI. There was a caption displayed below images before.

On some pages it was a text caption of what appears to be the image title:
- Timeline
  - https://bandhistory.unl.edu/timeline
  - https://cdrhrailsdev.unl.edu/bandhistory/timeline
- Sights & Sounds > Images
  - https://bandhistory.unl.edu/multimedia/images
  - https://cdrhrailsdev.unl.edu/bandhistory/multimedia/images
- Sights & Sounds > Audio > On Parade
  - https://bandhistory.unl.edu/multimedia/audio/on_parade
  - https://cdrhrailsdev.unl.edu/bandhistory/multimedia/audio/on_parade

On item pages, it was a link to view the image at a slightly higher resolution:
- https://bandhistory.unl.edu/item/band.doc.0002
- https://cdrhrailsdev.unl.edu/bandhistory/item/band.doc.0002

Plus is the alt-text of just "Full-size image" on all these images really better for a11y? I guess it's better than nothing, but losing some helpful UI for this feels icky 😬

**Comment by techgique** [comment]
I pushed another commit and the changes are live on the dev site. @erinchambers , please check if the caption work is good for a11y checks and if my SCSS is okay. I used some `!important`s and I'm skeptical that's the ideal approach or what you'd prefer.

**Comment by erinchambers** [comment]
It looks like we're missing some controls—I only see 2 buttons, zoom and exit, but there should be 4 (zoom, play, gallery, exit). Maybe a typo in the code? Here are some screenshots showing the difference between [Band History](https://cdrhrailsdev.unl.edu/bandhistory/item/band.doc.0002) and the [Cody Archive](https://cdrhrailsdev.unl.edu/codyarchive/item/wfc.bks00008#gallery-1)

<img width="309" height="368" alt="Band History screenshot with 2 controls" src="https://github.com/user-attachments/assets/be97eb4d-17f4-4281-b959-edbca745ca35" />
<img width="255" height="368" alt="Image" src="https://github.com/user-attachments/assets/1941b23f-873a-452a-b393-f335b557d7d8" />

**Comment by erinchambers** [comment]
@techgique just noticed that the controls look correct on the Timeline (https://cdrhrailsdev.unl.edu/bandhistory/timeline), just not on the items with multiple images (example: https://cdrhrailsdev.unl.edu/bandhistory/item/band.doc.0002)
